README.md

```diff
@@ -693,6 +690,49 @@ Updating virtual Go repository:
 err = servicesManager.UpdateVirtualRepository().Go(params)
 ```
 
+#### Creating and Updating Federated Repository
```

Comment:
Let's add all the new README section to the table of content as well.

Reply:
Oh yes, I missed that. I've updated the table of contents but just thinking, now that GitHub has a table of contents built in now, this is probably no longer needed and reduces some maintenance on the README section.